app/src/main/res/layout-sw360dp/dialog_edit_comment.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
App will crash for devices with a display smaller width of 360 dp. Maybe this could go on the layout/ folder instead

Reply:
Hm not sure how that happened. Not intentional! Good catch. :)